.github/README.md

```diff
@@ -0,0 +1,293 @@
+# Kedro-Viz
```

Comment:
hey @jitu5 , I'm wondering why there are 2 readme files, one is in here, and the other is in the root level [here](https://github.com/kedro-org/kedro-viz/blob/a935cc13e83420c04ce41f05c001ed3089d216b2/README.md)

Quickly go through them, they're kind of similar but written slightly differently, why is that a case?

Reply:
as in i can see your description but I'm not sure why we decided to have 2 separate readme files like that? 

```
README.md Relocation: Existing README.md file has been moved to the .github folder.
NPM-Specific README.md: The existing README.md has been modified to be used specifically for NPM.
```

Reply:
@Huongg To explain why we are making two README.md files I added reference of my personal Github repository of Kedro-viz 

> Github: https://github.com/jitu5/kedro-viz
> NPM: https://www.npmjs.com/package/kedro-vizzz
> PyPI: https://test.pypi.org/project/kedro-vizzz/9.0.15

In our package listed on [npm](https://www.npmjs.com/package/@quantumblack/kedro-viz), the README.md is tailored specifically to the React component part of Kedro-Viz not anything about python. This README includes code snippets showcasing all the properties of the component, explanations of each property, and a demo example, specifically found in our kedro-viz-standalone repository.

Due to the constraints of npm, which requires the README.md to be in the root directory to be properly displayed on the package page, I have arranged for the existing README.md file to be npm-specific. On the other hand, GitHub allows the README.md to be located in several places, including the .github, root, or docs directory as per [GitHub's guidelines on README files](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#about-readmes). This setup enables us to maintain separate README.md files for npm and GitHub/PyPI